pilot/pkg/xds/pushqueue.go

```diff
@@ -56,12 +50,6 @@ func (p *PushQueue) Enqueue(con *Connection, pushRequest *model.PushRequest) {
 		return
 	}
 
-	// If its already in progress, merge the info and return
-	if request, f := p.processing[con]; f {
```

Comment:
This can not happen.  The uint tests have all passed after removing the code.

Reply:
Would this even create problem if we merge in progress PushRequest as we do not know in which stage it is ? Not sure why it was added in the first place?